migrate_cpp/migrate_cpp.py

```diff
@@ -50,7 +50,7 @@ def run(self) -> None:
             self._print_header("Done!")
         except subprocess.CalledProcessError as e:
             # Discard the stack for subprocess errors.
-            sys.exit(e)
+            sys.exit(1)
```

Comment:
```suggestion
            sys.exit(str(e))
```

Note that [sys.exit](https://docs.python.org/3/library/sys.html#sys.exit) permits passing objects to print them, so `sys.exit(1)` removes printing the error message. I think this will address the mypy error.